fdbclient/NativeAPI.actor.cpp

```diff
@@ -3003,6 +3003,22 @@ void Transaction::setOption( FDBTransactionOptions::Option option, Optional<Stri
 			if(trLogInfo) {
 				trLogInfo->maxFieldLength = options.maxTransactionLoggingFieldLength;
 			}
+			if (info.debugID.present()) {
+				TraceEvent(SevInfo, "TransactionBeingTraced")
+					.detail("DebugTransactionID", trLogInfo->identifier)
+					.detail("ServerTraceID", info.debugID.get().toString());
+
+			}
```

Comment:
Was this meant to go in the `LOG_TRANSACTION` option?

Reply:
Correct. Fixed.